examples/audio_out.c

```diff
@@ -348,7 +348,13 @@ opensoundsys_open (int channels, int samplerate)
 		exit (1) ;
 		} ;
 
-	fmt = CPU_IS_BIG_ENDIAN ? AFMT_S16_BE : AFMT_S16_LE ;
+	fmt =
+#ifdef WORDS_BIGENDIAN
+	AFMT_S16_BE
+#else
+	AFMT_S16_LE
+#endif
```

Comment:
Not keen on this change. Personally I consider `AC_C_BIGENDIAN` hugely flawed. In the past is has cost me a couple of hours debugging and I have not forgiven it.

The problem that macro is that if defines `WORDS_BIGENDIAN` and the suggested usage is as you have used it here. The problem is that if you use it like this and forget to include `<config.h>` it will *sliently* do the wrong thing on big endian platforms which are far less tested anyway.